app/assets/stylesheets/application.scss

```diff
@@ -236,6 +235,7 @@ table {
 
   tr td:last-child {
     .btn {
+      font-size: 13px;
```

Comment:
Another problem with this is that most other text sizes have to be changed to match. Notice how the title is smaller than the regular font if we override it to 14px and by using the default it's just fine. 

![screenshot from 2015-12-09 10-53-26](https://cloud.githubusercontent.com/assets/598891/11682174/8b7a048a-9e63-11e5-8ae5-f86f71a12ccb.png)
![screenshot from 2015-12-09 10-53-53](https://cloud.githubusercontent.com/assets/598891/11682175/8da4f080-9e63-11e5-8dba-3013e02f3cec.png)